signer/src/block_observer.rs

```diff
@@ -436,11 +440,16 @@ impl<C: Context, B> BlockObserver<C, B> {
                 continue;
             }
 
+            let txid = tx.compute_txid();
+            if tx.is_coinbase() {
+                tracing::warn!(%txid, "ignoring coinbase tx when extracting sbtc transaction");
+                continue;
+            }
```

Comment:
nit: computing `txid` after this check can save us some cpu cycles.

Reply:
We would still need it for the logging though, so it should be equivalent

Reply:
of, I see, we use txid for logging, sorry